.github/workflows/workflow.yml

```diff
@@ -211,6 +211,8 @@ jobs:
         brew install protobuf
     - if: matrix.os == 'windows-latest'
       name: Install dependencies (windows-latest)
+      env:
+        ACTIONS_ALLOW_UNSECURE_COMMANDS: true
```

Comment:
From a GH pipeline output (https://pipelines.actions.githubusercontent.com/7sDcrH6ORgsV46IrjyGkkm9svY0T5k5MHqINOHfMTGdQaFcLxW/_apis/pipelines/1/runs/5432/signedlogcontent/66?urlExpires=2020-11-17T09%3A27%3A18.0737144Z&urlSigningMethod=HMACV1&urlSignature=oDQkk4wRl0waEP0wnQHvMbXHEZEPqyvpmEdkxhKgn5U%3D):

```
2020-11-17T09:09:24.3667593Z ##[error]Unable to process command '::add-path::C:\Program Files (x86)\Windows Kits\10\bin\10.0.18362.0\x64' successfully.
2020-11-17T09:09:24.3698299Z ##[error]The `add-path` command is disabled. Please upgrade to using Environment Files or opt into unsecure command execution by setting the `ACTIONS_ALLOW_UNSECURE_COMMANDS` environment variable to `true`. For more information see: https://github.blog/changelog/2020-10-01-github-actions-deprecating-set-env-and-add-path-commands/
```

https://github.blog/changelog/2020-10-01-github-actions-deprecating-set-env-and-add-path-commands/